hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -44,28 +44,46 @@
             description = "Table name")
   private String tableName;
 
+  @CommandLine.Option(names = {"--withKey"},
```

Comment:
NIT: as I am an oldschool command line user, I would prefer to use `--with-keys` instead of camel case

Reply:
Sure, will do.